docs/docs/daft.md

```diff
@@ -125,21 +125,21 @@ Daft and Iceberg have compatible type systems. Here are how types are converted
-| `fixed(L)` | [`daft.DataType.binary()`](https://www.getdaft.io/projects/docs/en/latest/api_docs/datatype.html#daft.DataType.binary) |
-| `binary` | [`daft.DataType.binary()`](https://www.getdaft.io/projects/docs/en/latest/api_docs/datatype.html#daft.DataType.binary) |
+| `boolean` | [`daft.DataType.bool()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.bool) |
+| `int` | [`daft.DataType.int32()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.int32) |
+| `long` | [`daft.DataType.int64()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.int64) |
+| `float` | [`daft.DataType.float32()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.float32) |
+| `double` | [`daft.DataType.float64()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.float64) |
+| `decimal(precision, scale)` | [`daft.DataType.decimal128(precision, scale)`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.decimal128) |
+| `date` | [`daft.DataType.date()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.date) |
+| `time` | [`daft.DataType.time(timeunit="us")`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.int64) |
+| `timestamp` | [`daft.DataType.timestamp(timeunit="us", timezone=None)`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.timestamp) |
+| `timestampz` | [`daft.DataType.timestamp(timeunit="us", timezone="UTC")`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.timestamp) |
+| `string` | [`daft.DataType.string()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.string) |
+| `uuid` | [`daft.DataType.binary()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.binary) |
+| `fixed(L)` | [`daft.DataType.binary()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.fixed_size_binary) |
+| `binary` | [`daft.DataType.binary()`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.binary) |
 | **Nested Types** |
-| `struct(**fields)` | [`daft.DataType.struct(**fields)`](https://www.getdaft.io/projects/docs/en/latest/api_docs/datatype.html#daft.DataType.struct) |
-| `list(child_type)` | [`daft.DataType.list(child_type)`](https://www.getdaft.io/projects/docs/en/latest/api_docs/datatype.html#daft.DataType.list) |
-| `map(K, V)` | [`daft.DataType.map(K, V)`](https://www.getdaft.io/projects/docs/en/latest/api_docs/datatype.html#daft.DataType.map) |
+| `struct(**fields)` | [`daft.DataType.struct(**fields)`](https://www.getdaft.io/projects/docs/en/latest/api/datatypes/#daft.datatype.DataType.struct) |
```

Comment:
Why does `fields` contain leading `**`? Is it for Python or Rust syntax?

Reply:
Python Kwargs - https://book.pythontips.com/en/latest/args_and_kwargs.html#usage-of-kwargs